### ISSUE TITLE
查找api帮助的正确方法是？

### ISSUE THREAD

**Issue by A-Pai** [PaddlePaddle/Paddle#31297]
查找api帮助的正确方法是？
感觉不好找：按照怎样的逻辑层层查？或者怎样搜索？
比如想查model.fit()，查不到啊！费力
![QQ截图20210301075149](https://user-images.githubusercontent.com/5112397/109438130-7187ac80-7a63-11eb-8ae9-185bd4d8726e.jpg)

**Comment by LielinJiang** [comment]
可以通过直接搜API的名称来查找。https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/hapi/model/Model_cn.html#fit-train-data-none-eval-data-none-batch-size-1-epochs-1-eval-freq-1-log-freq-10-save-dir-none-save-freq-1-verbose-2-drop-last-false-shuffle-true-num-workers-0-callbacks-none

**Comment by A-Pai** [comment]
got it，谢谢
